### ISSUE TITLE
It says virus detected in latest release so I can't download it

### ISSUE THREAD

**Issue by Farfighters** [rebinf/QuranVideoMaker#79]

**Comment by Farfighters** [comment]
![image](https://github.com/user-attachments/assets/ce7928fc-fe15-4c61-be46-a45a05f4ca2c)

**Comment by Farfighters** [comment]
Not sure why it's saying that then, will try in different browser and see inshallah

**Comment by Farfighters** [comment]
Even other browsers have the same problem?

**Comment by rebinf** [comment]
not sure, very little has changed since the previous release.
I've just updated some code docs and created another release, see if it says the same about the latest one.

**Comment by rebinf** [comment]
Zip url scan results from virus total:

https://www.virustotal.com/gui/url/a6473e85dbaa7c2fc0d24aa62a9ef256ea773998fa2f02cc48f78a5f4cd83ac8?nocache=1

Actual zip file results from virus total:

https://www.virustotal.com/gui/file/00cd46a8b495a594e5f3a936f01a5441140a3367b97bf606960f971d6b4666cd/detection

**Comment by rebinf** [comment]
glad it's working now.

**Comment by Farfighters** [comment]
Assalamualaikum wbt, for some reason the latest version is saying virus detected and not allowing me to download it again. I tried on multiple laptops (all Windows) and they all had the same problem. Whilst there might be a way for me to get around it, others will find it very hard, so is it possible to do something similar to what you did last time? It might be Windows trying to have better virus protection, I'm not sure (I only have Windows devices, so I don't know about other platforms). I also tested on multiple browsers and same issue came up.
